astropy/visualization/wcsaxes/tests/test_wcsapi.py

```diff
@@ -184,7 +184,7 @@ def world_to_array_index_values(self, *world_arrays):
     def world_axis_object_components(self):
         return [('freq', 0, 'value'),
                 ('time', 0, 'mjd'),
-                ('celestial', 0, 'spherical.lon.degree')
+                ('celestial', 0, 'spherical.lon.degree'),
```

Comment:
Wait a minute... how did this not fail CI?

Reply:
Because the property is never called 😅

Reply:
@astrofrog , looks like adding a test to cover this is in order?

Reply:
Test coverage fail....

Reply:
I'm not sure if this *really* needs a test since it's just a test-class that needs to satisfy the abstract properties of `BaseLowLevelWCS` that might not be even relevant for the test.

That's why I just fixed the SyntaxWarning.

Reply:
Given that the PR is merged now, should we open an issue for this issue?

Reply:
Indeed, @MSeifert04 ! Do you want the honor? :smile:

Reply:
okay

Reply:
See #9241